docs/changelog.md

```diff
@@ -41,11 +41,11 @@ Changed:
 
 Added:
 
-- [CookieboxCalibration][extra.recipes.CookieboxCalibration] to calibrate data
+- [CookieboxCalibration][extra.apps.CookieboxCalibration] to calibrate data
```

Comment:
These need to be updated:
```suggestion
- [CookieboxCalibration][extra.applications.CookieboxCalibration] to calibrate data
```

(if you run `mkdocs serve` it'll list all broken links)

Reply:
Thanks, fixed.